tests/e2e/support/objects/app-files/utils/editor.ts

```diff
@@ -4,7 +4,7 @@ const closeTextEditorOrViewerButton = '#app-top-bar-close'
 const saveTextEditorOrViewerButton = '#app-save-action'
 const texEditor = '#text-editor'
 const pdfViewer = '#pdf-viewer'
-const imageViewer = '#preview'
+const imageViewer = '.preview-controls-action-bar'
```

Comment:
```suggestion
const imageViewer = '.stage'
```

The `stage` selector will only appear if loading the content was successful.

Reply:
thanks 😍